Test/git-issues/git-issue-3187.dfy.expect

```diff
@@ -1,3 +1,5 @@
-git-issue-3187.dfy(6,31): Error: no comma is allowed between provides and reveals clauses in an export declaration
-git-issue-3187.dfy(6,31): Error: rbrace expected
-2 parse errors detected in git-issue-3187.dfy
+git-issue-3187.dfy(6,20): Error: no comma is allowed between provides and reveals and extends clauses in an export declaration
```

Comment:
Style: upper/lowercase and with/without full-stop (see also below).

Reply:
See previous response